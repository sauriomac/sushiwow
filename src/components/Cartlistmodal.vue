<template>
  <div>
    <Cartlist />
  </div>
</template>
<script>
import Cartlist from "@/views/CartList.vue";

export default {
  name: "Cartlistmodal",

  components: {
    Cartlist,
  },
};
</script>