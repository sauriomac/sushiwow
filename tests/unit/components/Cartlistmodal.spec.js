import { shallowMount, mount } from '@vue/test-utils'
import Cartlistmodal from '@/components/Cartlistmodal'


describe('Cartlistmodal',()=>{

    test('match with snapshot with Cartlistmodal',() => {

        const wrapper = shallowMount( Cartlistmodal)

        expect( wrapper.html() ).toMatchSnapshot()

    })




})