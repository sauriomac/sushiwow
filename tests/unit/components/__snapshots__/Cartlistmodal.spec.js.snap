// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cartlistmodal match with snapshot with Cartlistmodal 1`] = `
<div>
  <cartlist-stub></cartlist-stub>
</div>
`;
